tardis/montecarlo/montecarlo_numba/single_packet_loop.py

```diff
@@ -1,11 +1,12 @@
-from numba import njit
+from numba import njit, jit
```

Comment:
Is this import used?

Reply:
No. I will remove it. Was added when I was trying to test out a new implementation 😓